packages/lean_analytics/lib/firebase/firebase_analytics_widget.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import 'package:lean_analytics/lean_analytics.dart';
+import 'package:provider/provider.dart';
+
+class FirebaseAnalyticsWidget extends StatelessWidget {
+  const FirebaseAnalyticsWidget({super.key, required this.child});
+
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    final analytics = FirebaseLeanAnalytics();
+
+    return Provider<LeanAnalytics>.value(
```

Comment:
Do we want to introduce implicit dependencies from a library?

Reply:
Ok, I removed the widget and provider from the app